src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -375,7 +379,7 @@
                :params {:environment env-name :code_id code-id}}
               (catch Exception e
                 (log/debug (i18n/trs "Static file unavailable for {0}: {1}" file e))
-                {:path (format "/puppet/v3/file_content/tasks/%s/%s" module-name base-name)
+                {:path (format "/puppet/v3/file_content/%s" static-path)
                  :params {:environment env-name}}))]
     {:filename base-name
      :sha256 sha256
```

Comment:
I think we also should update https://github.com/puppetlabs/puppetserver/blob/master/src/clj/puppetlabs/services/master/master_core.clj#L669-L670 to allow `lib` to be addressable as a static file endpoint.